static/app/views/replays/detail/issueList.tsx

```diff
@@ -54,8 +51,6 @@ function IssueList({projectId, replayId}: Props) {
         `/organizations/${organization.slug}/issues/`,
         {
           query: {
-            // TODO(replays): What about backend issues?
-            project: projectId,
```

Comment:
@JoshFerge pretty sure we're gonna run into the `global-views` gating here. 

I don't think i can ship this unless there's a way around this.

Reply:
fixed by: https://github.com/getsentry/sentry/pull/48493